utils/src/ooinstall/openshift_ansible.py

```diff
@@ -127,7 +127,7 @@ def default_facts(hosts, verbose=False):
 
 def run_main_playbook(hosts, hosts_to_run_on, verbose=False):
     global CFG
-    inventory_file = generate_inventory(hosts)
+    inventory_file = generate_inventory(hosts_to_run_on)
```

Comment:
Hmm, is this going to interfere with fact gathering on the hosts that aren't being installed on...  In particular, I'm thinking about the facts for oo_first_master.  Really, we should be handling running on a subset in the scaleup playbook.